src/Features/RoyalShaving/RoyalShaving.tsx

```diff
@@ -0,0 +1,346 @@
+import { ReactElement } from 'react';
+import Image from 'next/image';
+import { Layout } from '../../components/Layout';
+import { Header } from '../../components/Header';
+import { LinkOptions } from '../../types/link';
+import { ServicePromo } from '../../components/ServicePromo';
+import { ServiceAboutSection } from '../../components/ServiceAboutSection';
+import { StageHeader } from './components/StageHeader/StageHeader';
+import { StageSection } from './components/StageSection';
+import { StageContent } from './components/StageContent';
+
+interface HeaderPageProps {
```

Comment:
Props should have the exact same name as the component + Props postfix, so it is RoyalShavingProps

Reply:
My bad with inattentive copying...

Reply:
Done